lib/string_reverse.rb

```diff
@@ -1,6 +1,19 @@
 # A method to reverse a string in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n), where n is the number of iterations in the while loop
+# Space complexity: Constant or O(1), because the additional stirage needed is just the variable "a"

```

Comment:
Good explanation!

[nit] stirage -> storage. Don't forget you also use `i` and `j`.